sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,11 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self._eval_is_real():
```

Comment:
If that is ever extended to include all purely imaginary arguments, then this implementation would be wrong.